pom.xml

```diff
@@ -36,7 +36,7 @@
     <mods.directory>target/mods</mods.directory>
 
     <!--Dependency versions-->
-    <vertx.version>3.2.1</vertx.version>
+    <vertx.version>3.3.2</vertx.version>
```

Comment:
Great Julien. I have also just added a somewhat usable change to make clustering to work inside this. Isaiah doesn't seem to be interested in this project and not reachable. Perhaps I should post a link to vertx mailing list with my fork and suggest anyone who wants to try it out?